mittab/libs/data_import/import_rooms.py

```diff
@@ -33,6 +32,8 @@ def import_rooms(fileToImport):
     num_rooms = 0
     found_end = False
     room_errors = []
+
+    # TODO: Why is this even here? Why not just use the sh.nrows and sh.ncols -- Kevin
```

Comment:
🤷‍♂️ Not sure why but also not a huge deal that this is here. Try to leave stuff like this to GitHub issues or just leave it as-is. Too many comments can make the code harder to read imo